helpothers/templates/base.html

```diff
@@ -66,6 +67,12 @@
     <script src="{% static "js/jquery.easing.min.js" %}"></script>
     <script src="{% static "js/jquery.fittext.js" %}"></script>
     <script src="{% static "js/wow.min.js" %}"></script>
+    <script src="{% static "js/jquery-ui.min.js" %}"></script>
+    <script src="{% static "js/jspanel/jquery.jspanel.min.js" %}"></script>
```

Comment:
Ok. Best to remove it now indeed. 
Btw The project is using twitter bootstrap so if we're doing popups or toast messages it's better to use that.